### ISSUE TITLE
Error When Logging in to Tableau Online

### ISSUE THREAD

**Issue by andyoneal** [tableau/tabcmd#158]
Running tabcmd v2.0.5 on macOS 12.4.

I'm getting a confusing error when trying to login to Tableau Online. If I follow the docs and use "https://online.tableau.com" as the server, I get an error. If I use the exact same command, but change the server to "https://prod-useast-a.online.tableau.com", everything works as expected.

Server = https://online.tableau.com
```
DEBUG: src.execution.tabcmd_controller: 29: Namespace(certificate=None, no_certcheck=False, continue_if_exists=True, no_cookie=False, logging_level='DEBUG', no_prompt=False, token_name=None, username='xxxxx@xxxxx.com', token=None, password='xxxxx', password_file=None, proxy=None, no_proxy=True, server='https://online.tableau.com', site_name='xxxxx', timeout=None, language=None, country=None, func=<class 'src.commands.auth.login_command.LoginCommand'>)
DEBUG: LoginCommand: 23: ======================= Launching command =======================
INFO:  Creating new session
INFO:  =====   Server: https://online.tableau.com
INFO:  =====   Username: andy.oneal@netsparker.com
INFO:  Tableau Server Site: invicti
INFO:  Connecting to the server...
INFO:

	405000: Method Not Allowed
		The HTTP method 'GET' is not supported for the given resource
```

Server = https://prod-useast-a.online.tableau.com
```
DEBUG: src.execution.tabcmd_controller: 29: Namespace(certificate=None, no_certcheck=False, continue_if_exists=True, no_cookie=False, logging_level='DEBUG', no_prompt=False, token_name=None, username='xxxxx@xxxxx.com', token=None, password='xxxxx', password_file=None, proxy=None, no_proxy=True, server='https://prod-useast-a.online.tableau.com', site_name='xxxxx', timeout=None, language=None, country=None, func=<class 'src.commands.auth.login_command.LoginCommand'>)
DEBUG: LoginCommand: 23: ======================= Launching command =======================
INFO:  Creating new session
INFO:  =====   Server: https://prod-useast-a.online.tableau.com
INFO:  =====   Username: andy.oneal@netsparker.com
INFO:  Tableau Server Site: invicti
INFO:  Connecting to the server...
INFO:  Succeeded
```

**Comment by bcantoni** [comment]
Thanks @andyoneal - I'm pretty sure your conclusion there is the correct one (that you need to use your actual Tableau Online hostname). I'll mark this for further investigation and updating the docs if that's the correct solution here.

**Comment by jacalata** [comment]
This should get fixed by https://github.com/tableau/server-client-python/pull/1125

**Comment by jacalata** [comment]
This is fixed in 2.0.7.
